content/developer/reference/user_interface/views/include/list.rst

```diff
@@ -0,0 +1,569 @@
+.. _reference/user_interface/views/list:
+
+List
+====
+
+.. container:: row
+
+  .. code-block:: xml
+    :class: col-xxl-6
+
+    <tree>
+      ...
+    </tree>
+
+  .. image:: views/list.svg
+    :class: col-xxl-6
+
+The root element of list views is ``<tree>``\ [#treehistory]_. The list view's
+root can have the following attributes_:
+
+:string:
+  string_
+
+  View name
+
+:create:
+  boolean_
+
+  Disable/enable record creation on the view.
+
+:edit:
+  boolean_
+
+  Disable/enable record editing on the view.
+
+  If the ``edit`` attribute is set to ``false``, the ``editable`` option will be ignored.
+
+:delete:
+  boolean_
+
+  Disable/enable record deletion on the view through the **Action** dropdown.
+
+:import:
+  boolean_
+
+  Disable/enable record import data on the view.
+
+:export_xlsx:
+  boolean_
+
+  Disable/enable record export data on the view.
+
+:editable:
+  string_ chooses from ``top`` or ``bottom``
+
+  by default, selecting a list view's row opens the corresponding
+  :ref:`form view <reference/user_interface/views/form>`. The ``editable`` attributes
+  makes the list view itself editable in-place.
+
+  Valid values are ``top`` and ``bottom``, making *new* records appear respectively at
+  the top or bottom of the list.
+
+  The architecture for the inline :ref:`form view <reference/user_interface/views/form>`
+  is derived from the list view. Most attributes valid on a :ref:`form view
+  <reference/user_interface/views/form>`'s fields and buttons are thus accepted by list
+  views although they may not have any meaning if the list view is non-editable.
+
+  If the ``edit`` attribute is set to ``false``, the ``editable`` option **will be ignored**.
+
+:multi_edit:
+  ``1``
+
+  editable or not editable list can activate the multi-editing feature by defining the
+  `multi_edit="1"`
+
+:default_group_by:
+  string_ :ref:`model <reference/orm/model>` field name
+
+
+  whether the list view should be grouped if no grouping is specified via the action or
+  the current search. Should be the name of the field to group by when no grouping is
+  otherwise specified
+
+:default_order:
+  `Comma-separated values`_
+
+  overrides the ordering of the view, replacing the model's order (:attr:`-odoo.models.BaseModel._order` model attribute).
+  The value is a comma-separated list of :ref:`fields <reference/orm/fields>`, postfixed by ``desc`` to
+  sort in reverse order:
+
+  .. code-block:: xml
+
+    <tree default_order="sequence,name desc">
+      ...
+    </tree>
+
+:decoration-{$name}:
```

Comment:
```suggestion

:decoration-bf:
:decoration-it:
:decoration-danger:
:decoration-warning:
:decoration-info:
:decoration-muted:
:decoration-primary:
:decoration-success:
```

in this case I would specify them all (so it's easier to find) and explain just once what this does with the variations